amlip_docs/rst/demo/tensor_inference.rst

```diff
@@ -0,0 +1,144 @@
+
+
+Run Inference Node
+------------------
+
+In the second terminal, run the following command to process the inference:
+
+.. code-block:: bash
+
+    # Source colcon installation
+    source install/setup.bash
+
+    # To execute Inference Node with pre-trained model from TensorFlow
+    cd ~/AML-IP-demos-ws/src/AML-IP-demos/amlip_demos/amlip_demos
+    python3 inference_node.py tensorflow
+
+This execution expects an output similar to the one shown below:
+
+.. code-block:: bash
+
+    2023-02-14 14:50:42.711797: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2 FMA
+    To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
+    Inference Node AMLInferenceNode.7c.df.34.e8 ready.
+    Model Handle at TensorFlow Hub: /home/user/AML-IP/src/amlipdemos/amlip_demos/resource/tensorflow/models/centernet_hourglass_512x512_kpts_1
+    loading model...
+    WARNING:absl:Importing a function (__inference_batchnorm_layer_call_and_return_conditional_losses_42408) with ops with unsaved custom gradients. Will likely fail if a gradient is requested.
+    WARNING:absl:Importing a function (__inference_batchnorm_layer_call_and_return_conditional_losses_209416) with ops with unsaved custom gradients. Will likely fail if a gradient is requested.
+    WARNING:absl:Importing a function (__inference_batchnorm_layer_call_and_return_conditional_losses_220336) with ops with unsaved custom gradients. Will likely fail if a gradient is requested.
+    ...
+    WARNING:absl:Importing a function (__inference_batchnorm_layer_call_and_return_conditional_losses_55827) with ops with unsaved custom gradients. Will likely fail if a gradient is requested.
+    WARNING:absl:Importing a function (__inference_batchnorm_layer_call_and_return_conditional_losses_56488) with ops with unsaved custom gradients. Will likely fail if a gradient is requested.
+    model loaded!
+    Selected model:tensorflow
+    2023-02-14 14:51:14.165305: W tensorflow/core/grappler/optimizers/loop_optimizer.cc:907] Skipping loop optimization for Merge node with control input: StatefulPartitionedCall/cond/then/_918/cond/Assert_2/AssertGuard/branch_executed/_1123
+    inference ready!
+    sending inference: kite: 81% person: 76% person: 74% kite: 72% kite: 71% person: 67% kite: 67% kite: 66% kite: 59% person: 54% person: 52% person: 52% person: 52% person: 51% kite: 50% person: 45% person: 45% surfboard: 40% surfboard: 39% surfboard: 31% kite: 30% person: 24% bird: 23% handbag: 23% person: 22% surfboard: 21% kite: 20% handbag: 20% kite: 20% surfboard: 19% person: 18% person: 18% kite: 16% person: 15% person: 15% person: 14% person: 14% surfboard: 13% kite: 13% backpack: 12% person: 12% kite: 11% person: 11% surfboard: 11% surfboard: 11% kite: 11%
+    Inference sent to client AMLEdgeNode.e5.45.c7.f5.
```

Comment:
Add some other sections:

- `Run multiple nodes of each kind`. It is not needed to do the whole tutorial, but mention that it is possible to have multiple models, and to ask for multiple inferences in parallel
- `How to use your own model`

Reply:
A section is not a note.
The idea of this whole project is parallelize the process of training and inference. Please, elaborate this more